tests/nnstreamer_plugins/unittest_plugins.cc

```diff
@@ -1909,6 +1909,100 @@ TEST (testTensorTransform, arithmeticChangeOptionString)
   gst_harness_teardown (h);
 }
 
+/**
+ * @brief Test for tensor_transform arithmetic (flex tensor)
+ */
+TEST (testTensorTransform, arithmeticFlexTensor)
```

Comment:
negative cases?

Reply:
After changing caps string (about tensor format)
All elements with flex caps should have formal condition to check data size. Now tensor-transform does not compare data size in transform().

Reply:
Ok. Let's postpone it until then.